balancer/endpointsharding/endpointsharding_test.go

```diff
@@ -142,9 +142,9 @@ func (s) TestEndpointShardingBasic(t *testing.T) {
 		ServiceConfig: sc,
 	})
 
-	cc, err := grpc.Dial(mr.Scheme()+":///", grpc.WithResolvers(mr), grpc.WithTransportCredentials(insecure.NewCredentials()))
+	cc, err := grpc.NewClient(mr.Scheme()+":///", grpc.WithResolvers(mr), grpc.WithTransportCredentials(insecure.NewCredentials()))
 	if err != nil {
-		log.Fatalf("Failed to dial: %v", err)
+		log.Fatalf("Failed to NewClient: %v", err)
```

Comment:
"dial" is a verb.  "NewClient" is not.  Please fix the errors that don't have proper grammar.

Reply:
@hanut19 `failed to create new client`. Similar to what you changed elsewhere. I think you missed this one